src/cascadia/TerminalSettingsEditor/Appearances.xaml

```diff
@@ -23,6 +23,12 @@
                           x:DataType="local:EnumEntry">
                 <RadioButton Content="{x:Bind EnumName, Mode=OneWay}" />
             </DataTemplate>
+            <DataTemplate x:Key="EnumComboBoxTemplate"
+                          x:DataType="local:EnumEntry">
+                <StackPanel Orientation="Horizontal">

```

Comment:
Do you even need the `StackPanel`? It seems like it only has one child so it's not really doing anything.

Reply:
This probably applies to the copied over version in the other files.